### ISSUE TITLE
cluster init should create /root/.ssh

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1159]
Originally reported of Google Code with ID 1103.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

# gnt-cluster --version
gnt-cluster (ganeti v2.14.0) 2.14.0

# gnt-cluster version
Software version: 2.14.0
Internode protocol: 2140000
Configuration format: 2140000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version v2.14.0

# hspace --version
hspace (ganeti) version v2.14.0
compiled with ghc 7.6
running on linux x86_64

<b>What distribution are you using?</b>
# cat /etc/*-release
SUSE Linux Enterprise Server 11 (x86_64)
VERSION = 11
PATCHLEVEL = 3
LSB_VERSION="core-2.0-noarch:core-3.2-noarch:core-4.0-noarch:core-2.0-x86_64:core-3.2-x86_64:core-4.0-x86_64"

<b>What steps will reproduce the problem?</b>
1. remove /root/.ssh if it exists
2. init a new cluster

I created a new cluster based on 2.14. While running "cluster init" I noticed, that ganeti complains about that /root/.ssh is missing:

gnt-cluster init ... mycluster.domain.tld 
Failure: command execution error:
Path /root/.ssh is not a directory

May be this can be handled by ganeti?

Thanks, Sascha.
```
Originally added on 2015-06-15 11:46:34 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-06-16 10:31:45 +0000 UTC.

Added Labels: Type-Enhancement Priority-Medium
Added to Milestone: Release2.14

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in master (= 2.16) with:

commit 68fed3b97594af2f5a41319884636389625f9adb
Author: Helga Velroyen <helgav@google.com>
Date:   Wed Jul 1 17:22:23 2015 +0200

    Create .ssh directory if it does not exist yet
    
    This patch extends the SSH handling by creating the .ssh
    directory when generating SSH keys if it did not exist
    yet. This fixes Issue 1103.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

```
Originally added on 2015-07-03 09:33:39 +0000 UTC.

Mentioned Issue with GoogleCode ID 1103 has been migrated to GitHub with ID #1159.
Changed State: Fixed
Added to Milestone: Release2.16
Removed from Milestone: Release2.14

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2016-01-11 15:04:51 +0000 UTC.

Changed State: Released
